src/privateer/restore.py

```diff
@@ -0,0 +1,71 @@
+import glob
```

Comment:
yeah, it seemed like a more natural split than having `schedule` as its own file, since the scheduling and manual backup share a bunch of code, whereas the restore functions are totally independent.